test/typeform-provider.test.js

```diff
@@ -41,7 +41,6 @@ describe('typeform-provider', () => {
   })
 
   test('form-basic', async () => {
-    if (!Config) return
```

Comment:
we actually do need - it makes the github action pass - we don't want to generate external requests from the github action